repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/identityv3/authz/IdentityV3AuthZBypassTest.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package features.filters.identityv3.authz
+
+import framework.ReposeValveTest
+import framework.mocks.MockIdentityV3Service
+import org.joda.time.DateTime
+import org.rackspace.deproxy.Deproxy
+import org.rackspace.deproxy.MessageChain
+import spock.lang.Unroll
+
+/**
+ * Created by adrian on 8/16/16.
+ */
+class IdentityV3AuthZBypassTest extends ReposeValveTest {
+    def static originEndpoint
+    def static identityEndpoint
+    def static targetPort
+
+    static MockIdentityV3Service fakeIdentityV3Service
+
+    def setupSpec() {
+        deproxy = new Deproxy()
+
+        def params = properties.defaultTemplateParams
+        repose.configurationProvider.applyConfigs("common", params)
+        repose.configurationProvider.applyConfigs("features/filters/identityv3/common", params)
+        repose.configurationProvider.applyConfigs("features/filters/identityv3/authz/bypass", params)
+        repose.start()
+
+        targetPort = properties.targetPort
+        originEndpoint = deproxy.addEndpoint(properties.targetPort, 'origin service')
+        fakeIdentityV3Service = new MockIdentityV3Service(properties.identityPort, properties.targetPort)
+        identityEndpoint = deproxy.addEndpoint(properties.identityPort,
+                'identity service', null, fakeIdentityV3Service.handler)
+        //targetPort = properties.targetPort
+    }
+
+    def cleanupSpec() {
```

Comment:
Correct. The method is not necessary. I've been cleaning them up as I come across them, but I missed this one. I have not made a story for this, but I will do that now. I guess that since the child starts it, previous Repose devs just wanted to locally clean up after themselves. Since the parent checks to see if the Deproxy container is still up before attempting to shut it down, there is no harm no foul. However, cleaning these up in one fell swoop is probably best.